apps/docs.blocksense.network/components/common/RadioGroup.tsx

```diff
@@ -0,0 +1,124 @@
+'use client';
+
+import React, {
+  createContext,
+  useContext,
+  ReactNode,
+  HTMLAttributes,
+  FieldsetHTMLAttributes,
+} from 'react';
+
+import { cn } from '@blocksense/ui/utils';
+import { Label } from '@blocksense/ui/Label';
+import { Input } from '@blocksense/ui/Input';
+
+type RadioGroupContextValue = {
+  value: string;
+  name: string;
+  onChange: (value: string) => void;
+};
+
+const RadioGroupContext = createContext<RadioGroupContextValue>(
+  {} as RadioGroupContextValue,
+);
+
+type RadioGroupProps = Omit<
+  FieldsetHTMLAttributes<HTMLFieldSetElement>,
+  'onChange'
+> & {
+  value: string;
+  name: string;
+  onChange: (value: string) => void;
+  ariaLabel: string;
+  children: ReactNode;
+};
+
+export const RadioGroup = ({
+  value,
+  name,
+  onChange,
+  ariaLabel,
+  children,
+  ...props
+}: RadioGroupProps) => {
+  return (
+    <RadioGroupContext.Provider value={{ value, name, onChange }}>
+      <fieldset
+        {...props}
+        aria-label={ariaLabel}
+        className={cn('radiogroup flex flex-col gap-1', props.className)}
+      >
```

Comment:
```suggestion
      <fieldset
        aria-label={ariaLabel}
        className={cn('radiogroup flex flex-col gap-1', props.className)}
        {...props}
      >
```

Can you pass `{...props}` at the bottom of component props (in all components)